graceoncampus/goc-web-v3
# Create header and footer files

* Files have separate css files to have styling specific to the header/footer
* Banner behind the header is not included in this commit
* Add logo assets to public folder

Testing Steps
------------------------------------------------------------------------
1. yarn && yarn build && yarn start
2. navigate to localhost:3000
3. ensure that the attached images/mock matches up to the page
4. ...

Resources
------------------------------------------------------------------------
- react-bootstrap: https://react-bootstrap.github.io/
